lib/builder-webpack4/src/preview-preset.ts

```diff
@@ -1,17 +1,17 @@
-import webpackConfig from '../preview/iframe-webpack.config';
+import webpackConfig from './preview/iframe-webpack.config';
```

Comment:
I moved this file, because it's easier that way. I'd have to modify the pre-bundle script considerably to make it output files respecting the original directory structure.

It kinda makes sense to me to keep all bundled files together at the `src` directory.